framework/error.go

```diff
@@ -18,6 +21,19 @@ func IsInvalidConfig(err error) bool {
 	return microerror.Cause(err) == invalidConfigError
 }
 
+var invalidResourceVersionError = errors.StatusError{
```

Comment:
I thought this was a decent way to match this error, let me know if this can be improved.